rails-i18n.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = '>= 1.8.11'
 
   s.add_runtime_dependency('i18n', '>= 0.7', '< 2')
-  s.add_runtime_dependency('railties', '>= 6.0.0', '< 7')
+  s.add_runtime_dependency('railties', '>= 6.0.0', '< 8')
```

Comment:
Do we really need to assert an incompatibility with a version that does not even exist?  End users are entirely capable of pinning and making these decisions themselves. 

It is a PITA when gems do this because every two years we have to this same exact dance

Reply:
I guess the reasoning is that we do not understand what will change in Rails in the future. Then I think it is better that the gem authors can decide: "This is now supported by Rails 8", instead of all the users updating automatically and then get mad when it does not work.

But when it is not maintained actively it is a problem.